mezzanine/pages/tests.py

```diff
@@ -31,6 +31,16 @@
 
 class PagesTests(TestCase):
 
+    def setUp(self):
+        """
+        Make sure we have a thread-local request with a site_id attribute set.
+        """
+        super(PagesTests, self).setUp()
+        from mezzanine.core.request import _thread_local
+        request = self._request_factory.get('/')
+        request.site_id = settings.SITE_ID
+        _thread_local.request = request
+
```

Comment:
This could alternatively go in `TestCase` to ensure that `current_request()` works everywhere, and we could just set the `site_id` attribute here.